Formula/chapel.rb

```diff
@@ -39,24 +60,14 @@ def install
     # Must be built from within CHPL_HOME to prevent build bugs.
     # https://github.com/Homebrew/legacy-homebrew/pull/35166
     cd libexec do
+      # TODO: when updating to llvm@13 or later, add `CHPL_LLVM_GCC_PREFIX=none`
+      # since the llvm@13 package should be configured to use a reasonable GCC
```

Comment:
No idea if `CHPL_LLVM_GCC_PREFIX=none` should be added here. Only left as comment since it previously mentioned `llvm@13` which is built with `gcc` while `llvm@12` is built with system `gcc-5`.